docs/releases/creating-a-release.md

```diff
@@ -64,75 +81,51 @@ graph LR
 
-#### Workflow details
-
-This section describes what the workflow above does. Understanding the workflow actions is useful in case the workflow fails.
-
-1. Create a Pull Request to merge the prerelease branch back into the base branch.
-    1. For stable release only: Before creating the Pull Request, @azure/communiation-react will be version bumped once again to a `-beta.0` version (this ensures that we can continue to add changes of `prerelease`) type.
-1. Create a new release branch off of the prerelease branch. This branch will be used for the eventual release, but **it will never be merged back in the base branch**.
-  1. On the release branch will check what kind of release this is, beta-release-major, beta-release-minor, beta-release-patch , beta-release-beta, stable-major, stable-minor, stable-patch or stable-remove-beta-suffix based on the selection here it will update the sdk versions.
-  2. The action will synchronize the package telemetry verions on the release branch.
-  3. After making these changes it will make a PR from the pre-release branch back into `main`.
-
-Continuing the example above, this action should be triggered once `groom_changelog_3.0.0` is merged. It will create the following new branches ~~and Pull Requests~~ (Pull requests must now be created manually per [security policy](https://aka.ms/opensource/actions-changes-230217)):
-
-```mermaid
-graph LR
-  main[branch: main]
-  prerelease[branch: prerelease/3.3.0<br/>- bump version to 3.3.1-beta.0]
-  release[branch:release/3.0.0<br/>- Update dependency versions<br/>- Potential cherry-picks<br/>- Merge string translations<br/> Abandon branch once released]
+## Step 2: Prepare for release
 
-  main -.-|Created in step 1| prerelease
-  prerelease -->|Create Branch| release
-  prerelease -.-o|Create Pull Request| main
-```
+### Step 2.1 (Stable / Large Beta Only): API approval from Azure REST API stewardship board]
 
-After running the action, the beta checks on the CI should be automatically disabled if this is a stable release. Inversely, the stable checks should be automatically disabled if this is a beta relase. This is to ensure that any cherry pick PR's going into release dont run against the wrong build flavor in CI causing failures. If the wrong build flavor checks were not automatically disabled, edit the [CI workflow](../../.github/workflows/ci.yml): Modify the output `JSON` string to remove the `beta` flavor for `stable` releases / `stable` flavor for `beta` releases.
+For stable releases, or for "large" beta releases, we must get any API changes approved by the Azure REST API Stewardship board. See the [internal documentation](https://skype.visualstudio.com/SPOOL/_wiki/wikis/SPOOL.wiki/27654/Scheduling-an-Azure-Review-Board-%28ARB%29-Review) for how to reach out to the API stewardship board.
```

Comment:
Is the requirement for large beta's something from ARB or are we making that determination ourselves? if this is something that can get a beta blocked for a while maybe check with Alex and Wayne to make sure this is something we want.